File: impl/ruby/examples/basic.rb
$LOAD_PATH.unshift(File.expand_path("../lib", __dir__))
require "json"
require "makrell/formats"

puts JSON.pretty_generate(
  Makrell::Formats.parse_mron_string(
    'name Makrell features [comments "trailing commas" typed_scalars] stable false'
  )
)

File: impl/perl/examples/basic.pl
use strict;
use warnings;
use FindBin;
use lib "$FindBin::Bin/../lib";
use Makrell::Formats qw(parse_mron_string write_mron_string);

my $doc = parse_mron_string(q{name Makrell features [comments "trailing-commas" "typed-scalars"] stable false});
print write_mron_string($doc), "\n";
